<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QWidget" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>20</y>
     <width>341</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_repant">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>430</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>悔棋</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_newGame">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>430</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>新游戏</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ending">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>430</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>结束游戏</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_save">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>500</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>保存游戏</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_load">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>500</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>加载游戏</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_setting">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>500</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_about">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>570</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_aboutQt">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>570</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>关于QT</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
